grype/presenter/cyclonedx/vulnerability_test.go

```diff
@@ -0,0 +1,51 @@
+package cyclonedx
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCvssVersionToMethod(t *testing.T) {
```

Comment:
nit: mind adding a case to cover v2?

Reply:
adjusted in https://github.com/anchore/grype/pull/317/commits/528d45eaa973efc85357734adbfa5588dac6dde8